app/models/card.rb

```diff
@@ -7,6 +7,12 @@ def validate(record)
 end
 
 class Card < ActiveRecord::Base
-	validates :original_text,:translated_text, presence: true
-	validates_with CompareValidator
+  before_validation :define_review_date
+  validates :original_text,:translated_text, presence: true
+  validates_with CompareValidator
+
+  protected
+  def define_review_date
+    self.review_date = Time.now + 3.days
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.